templates/resources/xl-release/bin/run-in-container.sh

```diff
@@ -61,21 +112,22 @@ if [ ! -f "${APP_HOME}/conf/xl-release-server.conf" ]; then
       REPOSITORY_KEYSTORE_PASSPHRASE=`pwgen 16`
       echo "... Generating repository keystore passphrase: ${REPOSITORY_KEYSTORE_PASSPHRASE}"
     fi
+
     echo "... Generating repository keystore"
     keytool -genseckey -alias deployit-passsword-key -keyalg aes -keysize 128 -keypass "deployit" -keystore ${APP_HOME}/conf/repository-keystore.jceks -storetype jceks -storepass ${REPOSITORY_KEYSTORE_PASSPHRASE}
 
     echo "... Generating xl-release-server.conf"
     sed -e "s/\${ADMIN_PASSWORD}/${ADMIN_PASSWORD}/g" -e "s/\${REPOSITORY_KEYSTORE_PASSPHRASE}/${REPOSITORY_KEYSTORE_PASSPHRASE}/g" ${APP_HOME}/conf/xl-release-server.conf.template > ${APP_HOME}/conf/xl-release-server.conf
+    if [ ! "${SERVER_URL}" = "" ]; then
+      cat "server.url=${SERVER_URL}" >> ${APP_HOME}/conf/xl-release-server.conf
+    fi
 
     echo "Done"
   fi
 fi
 
-
-
-# Generate node specific configuration with IP address of container
-IP_ADDRESS=$(hostname -i)
```

Comment:
Aha! Now I remember why there should be two configs:
- The node-conf will be re-generated every time the container starts because the IP and hostname will be different.
- The new bits of config you've added should _not_ change every time. Or should they?